### PR TITLE
SDK-359 Integrated phpbench command

### PR DESCRIPTION
**Pull request by miroshnikov-spryker**

### CODE REVIEW

File: src/Extension/Tasks/PhpBenchTask.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+id: "validation:php:benchmark"
+short_description: "Check your code efficiency by running a benchmark"
+help: ~
+stage: build
+version: 0.1.0
+command: "php %sdk_dir%/vendor/bin/phpbench run %tests% --bootstrap=%bootstrap% %iterations% %revs% %report% %time-unit%"
```
Comment (vol4onok) [inline src/Extension/Tasks/PhpBenchTask.yaml:7]: if has default value use it like optional option

Comment (miroshnikov-spryker) [reply src/Extension/Tasks/PhpBenchTask.yaml:7]: I decided to remove default paths to not make unnecessary assumptions

Comment (vol4onok) [reply src/Extension/Tasks/PhpBenchTask.yaml:7]: is it an optional parameter for tool or required?

**Review by vol4onok — APPROVED** [review]

**Review by ScarIetSky — APPROVED** [review]

File: src/Extension/Tasks/PhpBenchTask.yaml (hunk 1 above)
Comment (miroshnikov-spryker) [reply src/Extension/Tasks/PhpBenchTask.yaml]: This parameters are required but can be provided via configuration file phpbench.json in project

File: src/Extension/Tasks/PhpBenchTask.yaml [hunk 2]
```diff
@@ -4,13 +4,21 @@ short_description: "Check your code efficiency by running a benchmark"
 help: ~
 stage: build
 version: 0.1.0
-command: "php %sdk_dir%/vendor/bin/phpbench run %tests% --bootstrap=%bootstrap% %iterations% %revs% %report% %time-unit%"
+command: "php %sdk_dir%/vendor/bin/phpbench run %tests% --bootstrap=%bootstrap% %group% %iterations% %revs% %report% %time-unit% --dump-file=%reportDir%/phpbench.benchmark.xml"
```
Comment (vol4onok) [inline src/Extension/Tasks/PhpBenchTask.yaml:7]: ReportConverter?

Comment (miroshnikov-spryker) [reply src/Extension/Tasks/PhpBenchTask.yaml:7]: Added report converter

**Review by vol4onok — APPROVED** [review]

**Pull request merged**